_posts/2020-02-06-xcode-selection.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: posts
+title: Changes to Xcode selection in Bazel
+authors:
+  - susinmotion
+---
+
+One of our major projects in the last year has been improving build times for interactive iOS builds by adding support for the Google-internal remote execution service from Mac. Combined with [dynamic execution]({% post_url 2019-02-01-dynamic-spawn-scheduler %}), which helps mitigate deficiencies in our remote Mac executors by running some actions locally, we are now able to deliver the best possible clean and incremental build times.
```

Comment:
Do we have to say "google internal" or can you skip that and just talk about remote execution for mac?

Reply:
Haven't we already had Bazel Mac remote execution capabilities for awhile?

Reply:
Changed first sentence to be Bazel-focused, and moved previous intro to "setting the scene"